.github/workflows/deploy-development-docker-image.yml

```diff
@@ -0,0 +1,56 @@
+name: Create and publish development Docker image
+
+on:
+  push:
+    branches-ignore:
+      - "main"
+  workflow_dispatch:
+
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: ${{ github.repository }}
+
+jobs:
+  build-and-push-image:
+    runs-on: self-hosted
+    permissions:
+      contents: read
+      packages: write
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Build project
+        uses: game-ci/unity-builder@v2
+        env:
+          UNITY_LICENSE: ${{ secrets.UNITY_LICENSE }}
+          UNITY_EMAIL: ${{ secrets.UNITY_EMAIL }}
+          UNITY_PASSWORD: ${{ secrets.UNITY_PASSWORD }}
+        with:
+          targetPlatform: WebGL
+          buildsPath: builds
+          customImage: 'unityci/editor:2022.1.6f1-webgl-1'
```

Comment:
`2022.1.6f1`?
Isn't our unity version `2021.3.2f1` at the moment?

Reply:
Yes, but we used always this version to build cause it was the only one working. Maybe this changed but at the time we configured the Github action this was the case and it is also used for our other Github actions.